### ISSUE TITLE
Missing stop times when timepoint=true, stop_times.txt contains values

### ISSUE THREAD

**Issue by evantrillium** [BlinkTagInc/gtfs-to-html#64]
@brendannee - Table 2169 is not pulling in values for route 1026, direction 1, for all stops past stop_id=757545

Here are the outputs and GTFS file and config file.
https://drive.google.com/drive/folders/1MktFfU9lusieKQ40O1wVcpoLyMbEtoGp?usp=sharing

Do you spot where the issue in the files is? I'm stumped

**Comment by brendannee** [comment]
I checked into this - it took me a while to figure it out.

In timetable_store_order.txt there was a stop in Graton stop_id=757552 near the end of the route, but all trips also stop at this stop right before stop_id=757553. Because this stop shows up more than once and was specified in `timetable_stop_order.txt`, it confused the function that sorts stoptimes into stops. I was able to get all of the trips to show up for timetable_id=2168 by removing the following line from `timetable_stop_order.txt`

`2168,757552,15,Graton Rd & Sonoma Ave,"20/Russian River Area, Forestville, Sebastopol, Santa Rosa"`

Note that ou don't have to specify stops in timetable_stop_order.txt - GTFS-to-HTML does a pretty good job of . guessing the order you want. For this route, it seems to work fine if you remove all lines for timetable 2168 in `timetable_stop_order.txt` - see my attached version.
[timetable_stop_order.txt](https://github.com/BlinkTagInc/gtfs-to-html/files/3269555/timetable_stop_order.txt)
